prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/health-auth-org-info-page/health-auth-org-info-page.component.ts

```diff
@@ -24,15 +26,20 @@ export class HealthAuthOrgInfoPageComponent implements OnInit {
    * has not yet entered their organization information.
    */
   public isInitial: boolean;
+  public upload: boolean;
```

Comment:
Can this be named something else?  On the other hand, hard to think of an alternative ... "hasClickedAddAgreement"?  🤔

Reply:
When i wrote this i was thinking the same thing. No idea what to call it. But can make it more explicit